spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email { 'example@email.com' }
```

Comment:
It needs to be genereted by [FFaker](https://github.com/ffaker/ffaker) gem. This column is uniqueness.